build_test/lib/src/copy_builder.dart

```diff
@@ -59,13 +55,8 @@ class CopyBuilder implements Builder {
     return {'': outputs};
   }
 
-  AssetId _copiedAssetId(AssetId inputId, int copyNum) {
-    var withExtension = inputId.addExtension(_copiedAssetExtension(copyNum));
-    if (outputPackage == null) {
-      return withExtension;
-    }
-    return new AssetId(outputPackage, withExtension.path);
```

Comment:
It used to be legal to do this with `declareOutputs` since we could take another package's source as input and output it into our own package.